src/core/index.js

```diff
@@ -33,7 +33,7 @@ export function registerAdapter(adapter) {
  * @param location Location to visit (a URL or path)
  * @param options Options to apply
  * @param options.action Type of history navigation to apply ("restore",
- * "replace" or "advance")
+ * "replace", "refresh" or "advance")
```

Comment:
I think the `refresh` action should be reserved for internal use, just as the `restore` action is.